project1/server/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-add_subdirectory(file_handler)
+add_subdirectory(file_handler server_tasks)
 
 add_library(server_lib server.cpp agent.cpp)
-target_link_libraries(server_lib wire_protocol file_handler)
+target_link_libraries(server_lib wire_protocol file_handler thread_pool)
 
-add_executable(server server_main.cpp)
+add_executable(server server_main.cpp server_tasks/nport_task.h server_tasks/nport_task.cpp server_tasks/agent_task.h server_tasks/agent_task.cpp server_tasks/tport_task.h server_tasks/tport_task.cpp server_tasks/command_ids.h server_tasks/command_ids.cpp server_tasks/server_task.h server_tasks/server_task.cpp)
```

Comment:
There's really no need to include `.h` files directly here. (Just told this to Nick also.)

Reply:
Gotcha. I think my CLion is set to do this by default every time a new `.h` or `.cpp` file is created...